scripts/update.sh

```diff
@@ -45,6 +45,9 @@ update_all(){
     if [ "$MOONRAKER_TELEGRAM_BOT_UPDATE_AVAIL" = "true" ]; then
       echo -e "|  ${cyan}● MoonrakerTelegramBot${default}                               |"
     fi
+    if [ "$MOONCORD_UPDATE_AVAIL" = "true" ]; then
```

Comment:
This will never be true.
See https://github.com/th33xitus/kiauh/pull/172/files#diff-2a3b76e3e38782eafbe940c261fd37c040694182130d22318fefd7f690fc5ed6R626 and please use `MOONCORD_UPDATE_AVAIL`